api/src/test/java/org/openmrs/module/aijar/RolePrivilegeImportTest.java

```diff
@@ -0,0 +1,18 @@
+package org.openmrs.module.aijar;
+
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.dataexchange.DataImporter;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class RolePrivilegeImportTest extends BaseModuleContextSensitiveTest {
+
+
+    protected static final String ROLE_PRIVILLEGE_DATASET_XML = "metadata/Role_Privilege.xml";
+
+    @Test
+    public void shouldImportWithOutAnyError() {
```

Comment:
@slubwama This test is not complete you need to add assertions 

1. The custom roles have been imported 
2. The count of the number of custom privileges expected
3. Specific privileges per role - which we know can cause trouble 

FYI - the roles are still not imported in my database

Reply:
hello @ssmusoke  two can always break when there is a change. which might lead to change the test as well. I have added tests that I think test the import and makes sure that they never break the test.